support-frontend/app/controllers/IdentityController.scala

```diff
@@ -57,9 +57,23 @@ class IdentityController(
       identityService
         .getUserType(email)
         .fold(
-          err => {
-            SafeLogger.error(scrub"Failed to retrieve user type for $email: ${err.toString}")
-            InternalServerError
+          _ match {
+            case GetUserTypeGotErrorResponse(response) =>
+              if (response.status >= 400 && response.status < 500) {
+                SafeLogger.info(s"4xx error when retrieving user type for $email: ${response.body}")
```

Comment:
Is it ok not to be scrubbing this info message here? I’ve changed it to info because it’s not an error, but I’m slightly worried that using `s""` instead of `scrub""` is an information leak.

Reply:
My concern here is that our logging level for application logs on AWS is `error`, hence why we scrub strings logged as errors. While this isn't technically an error, we need to think about whether it's something we want to have logged in production. I'd suggest keeping it as an error-type log message, and ensure we scrub the string.

Reply:
We discussed this on a video call: on reflection we’re pretty sure we’ve seen info-level logs in AWS, so this explanation isn’t quite right. However, it may be that we only _alert_ on levels higher than info, and so that’s why. I will check with others who may know more.

We also agreed that `warn` is probably a more correct error level for this situation than `info`, which I’ll change now.